### PR TITLE
class component section missing

### PR DESCRIPTION
**Pull request by ElviraQDP**
content needs to be revised to make a new version

### CODE REVIEW

File: src/content/lesson/making-react-components.es.md [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
+
+// como componente funcional
 function NavBar(props){
     return (<nav className="navbar navbar-light bg-light">
               <a className="navbar-brand" href="#">Navbar</a>
            </nav>);
 }
 
-//o como una class 
+//o como un componente de clae
```
Comment (alesanchezr) [inline src/content/lesson/making-react-components.es.md:28]: clae => clase

**Review by alesanchezr — CHANGES_REQUESTED** [review]

File: src/content/lesson/making-react-components.es.md [hunk 2]
```diff
@@ -1,29 +1,33 @@
...
+cover: "https://ucarecdn.com/84c4d84c-51b9-4906-a572-71cc07ecfc8c/"
 textColor: "white"
 date: "2020-10-19T12:36:31-04:00"
 tags: ["reactjs"]
-status: "draft"
+status: "published"
 
 ---
 
 ## En React.js todo es un `<Component />`
```
Comment (marcogonzalo) [inline src/content/lesson/making-react-components.es.md:12]:
```suggestion
## En React.js todo es un `<Componente />`
```

File: src/content/lesson/making-react-components.es.md [hunk 3]
```diff
@@ -1,29 +1,33 @@
...
 ***
 
-React.js separa tu código en pequeñas partes llamadas Componentes que pueden crearse / definirse como **class** o como **function**.  Cada componente es como una aplicación de React más pequeña que tiene su propia lógica y tiene un propósito final que es mostrar (Render) algo (por ejemplo: una barra de navegación de bootstrap, una lista desplegable, un modelo, un formulario dinámico, galería de imágenes, formulario de suscripción, casi todo puede diseñarse y codificarse como un componente React).
+React.js separa tu código en pequeñas partes llamadas Componentes que pueden crearse / definirse como **class** o como **function**.  Cada componente es como una aplicación de React más pequeña que tiene su propia lógica y tiene un propósito final que es renderizar algo (por ejemplo: un navbar de bootstrap, una dropdpwn list (lista desplegable), un modelo, un formulario dinámico, galería de imágenes, formulario de suscripción, casi todo puede diseñarse y codificarse como un componente React). Para hacer eso, el componente de React debe tener un `return` que retorne código JSX (HTML + JS anidado).
```
Comment (marcogonzalo) [inline src/content/lesson/making-react-components.es.md:15]:
```suggestion
React.js separa tu código en pequeñas partes llamadas Componentes que pueden crearse / definirse como **class** o como **function**.  Cada componente es como una aplicación de React más pequeña que tiene su propia lógica y tiene un propósito final que es renderizar algo. Por ejemplo: un `navbar` de bootstrap, una _dropdown list_ (lista desplegable), un modelo, un formulario dinámico, galería de imágenes, formulario de suscripción, casi todo puede diseñarse y codificarse como un componente React). Para hacer eso, el componente de React debe tener un `return` que devuelva código JSX (HTML + JS anidado).
```

File: src/content/lesson/making-react-components.es.md [hunk 4]
```diff
@@ -66,13 +83,17 @@ let menu = [
 <Navbar items={menu} logo="http://path/to/logo.png" />
 ```
 
-Ahora, dentro de `<Navbar />` podemos usar esos valores (que se entregan a través de la variable Props) para procesar la información proporcionada.
+Ahora, dentro de `<Navbar />` podemos usar esos valores (que se entregan a través de la variable Props) para renderizar la información proporcionada.
 
-Y, por último, debe indicar a React dónde debe renderizar ese componente en el DOM.
+Y, por último, deberías indicarle a React dónde debe renderizar ese componente en el DOM.
```
Comment (marcogonzalo) [inline src/content/lesson/making-react-components.es.md:88]:
```suggestion
Y, por último, debemos indicarle a React dónde debe renderizar ese componente en el DOM.
```

Comment (marcogonzalo) [reply src/content/lesson/making-react-components.es.md:88]: Venimos hablando en segunda persona plural

**Review by marcogonzalo — CHANGES_REQUESTED** [review]

**Review by alesanchezr — APPROVED** [review]

**Pull request merged**